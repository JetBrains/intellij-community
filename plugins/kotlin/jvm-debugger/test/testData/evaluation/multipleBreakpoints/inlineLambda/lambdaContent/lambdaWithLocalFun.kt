// IGNORE_K1

fun main() {
    foo {
        fun local(x : Int) = x * 2
        local(it)
    }
}

inline fun foo(block: (Int) -> Int) {
    // EXPRESSION: block(42)
    // RESULT: 84: I
    //Breakpoint!
    val x = 1
}