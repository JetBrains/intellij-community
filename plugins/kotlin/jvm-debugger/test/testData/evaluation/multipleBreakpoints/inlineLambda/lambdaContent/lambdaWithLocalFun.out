LineBreakpoint created at lambdaWithLocalFun.kt:14
Run Java
Connected to the target VM
lambdaWithLocalFun.kt:14
Compile bytecode for block(42)
Disconnected from the target VM

Process finished with exit code 0
